source/includes/fundamentals/code-examples/linq.cs

```diff
@@ -81,6 +81,11 @@ public class Ingredient
 
 // end-ingredient-model
 
+// start-nested-SelectMany
+var query = queryableCollection
+    .Select(r => r.Restaurants.SelectMany(r => r.Grades));
```

Comment:
I think here you wanted to use `.SelectMany`, instead of `.Select`

Reply:
I tried to make it a bit clearer that it's a document with a different schema.

As for the output, it's a little bit convoluted - the example is only printing out results for one document, and the way that we print on this page omits the outermost brackets since it prints each result separately. Therefore, while the output is technically the second, it looks like the first. I have edited the example a little bit to hopefully be a bit clearer.

Reply:
Thanks, looks good!